modules/core/core-elasticsearch-client/src/main/java/com/enonic/xp/elasticsearch/client/impl/EsClient.java

```diff
@@ -0,0 +1,253 @@
+    }
+
+    public RefreshResponse indicesRefresh( final RefreshRequest refreshRequest )
+    {
+        return wrap( () -> client.indices().refresh( refreshRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public CloseIndexResponse indicesClose( CloseIndexRequest closeIndexRequest )
+    {
+        return wrap( () -> client.indices().close( closeIndexRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public OpenIndexResponse indicesOpen( OpenIndexRequest openIndexRequest )
+    {
+        return wrap( () -> client.indices().open( openIndexRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public boolean indicesExists( GetIndexRequest getIndexRequest )
+    {
+        return wrap( () -> client.indices().exists( getIndexRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public GetResponse get( GetRequest getRequest )
+    {
+        return wrap( () -> client.get( getRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public MultiGetResponse mget( MultiGetRequest getRequest )
+    {
+        return wrap( () -> client.mget( getRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public BulkResponse bulk( BulkRequest bulkRequest )
+    {
+        return wrap( () -> client.bulk( bulkRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public SearchResponse scroll( SearchScrollRequest searchScrollRequest )
+    {
+        return wrap( () -> client.scroll( searchScrollRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public CountResponse count( CountRequest countRequest )
+    {
+        return wrap( () -> client.count( countRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public ClearScrollResponse clearScroll( ClearScrollRequest clearScrollRequest )
+    {
+        return wrap( () -> client.clearScroll( clearScrollRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public DeleteResponse delete( DeleteRequest deleteRequest )
+    {
+        return wrap( () -> client.delete( deleteRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public IndexResponse index( IndexRequest indexRequest )
+    {
+        return wrap( () -> client.index( indexRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public SearchResponse search( SearchRequest searchRequest )
+    {
+        return wrap( () -> client.search( searchRequest, RequestOptions.DEFAULT ) );
+    }
+
+    public static XContentBuilder jsonBuilder()
+    {
+        return wrap( XContentFactory::jsonBuilder );
+    }
+
+    private static <T> T wrap( final Callable<T> function )
+    {
+        final Thread currentThread = Thread.currentThread();
+        final ClassLoader contextClassLoader = currentThread.getContextClassLoader();
+        currentThread.setContextClassLoader( RestHighLevelClient.class.getClassLoader() );
+        try
+        {
+            return function.call();
+        }
+        catch ( IOException e )
+        {
+            throw new UncheckedIOException( e );
+        }
+        catch ( Exception e )
+        {
+            if ( e instanceof RuntimeException )
```

Comment:
That can be simplified `new RuntimeException(e)`

Reply:
Wrap RE into RE is unnecessary